js/Model.js

```diff
@@ -399,20 +399,18 @@ define(
 			resolveConceptSetExpressionSimple(expression, success) {
 				const callback = typeof success === 'function'
 					? success
-					: ({ data: info }) => {
-							this.conceptSetInclusionIdentifiers(info);
-							this.currentIncludedConceptIdentifierList(info.join(','));
-							this.conceptSetInclusionCount(info.length);
-							this.resolvingConceptSetExpression(false);
-						};
+					: ({ data }) => {
+						let info = data;
+						if (!Array.isArray(info)) {
+							throw new Error();
+						}
+						this.conceptSetInclusionIdentifiers(info);
+						this.currentIncludedConceptIdentifierList(info.join(','));
+						this.conceptSetInclusionCount(info.length);
+					};
 				this.resolvingConceptSetExpression(true);
-				const resolvingPromise = httpService.doPost(sharedState.vocabularyUrl() + 'resolveConceptSetExpression', expression);
-				resolvingPromise.then(callback);
-				resolvingPromise.catch((err) => {
```

Comment:
What about direct calls of the method? e.g. from `Application.js`. Why don't you leave the `catch`?